app/views/AutoAdjustView.js

```diff
@@ -0,0 +1,98 @@
+var Template = require('../util/Template');
+var Logger = require('../util/Logger');
+var Commands = require('../models/Commands');
+
+module.exports = function (Marionette) {
+
+    return Marionette.ItemView.extend({
+        template: Template('AutoAdjustView'),
+        className: 'gainsAdjustView', 
+
+        ui: { 
+
+            all_button: "#sendall",
+            rc_button: "#fullRC",
+            auto_button: "fullAuto",
+            ground_button: "fullGround",
+            pitch_select: "#pitchsource",
+            roll_select: "#rollsource",
+            head_select: "#headsource",
+            alt_select: "#altsource",
+            throttle_select: "#throttlesource",
+            flap_select: "#flapssource",
+            rolltype_select: "#rolltype",
+            pitchtype_select: "#pitchtype"
+
+        },
+     
+        events: {
+            "click #sendall": "sendall",
+            "click #fullRC": "fullRC",
+            "click #fullAuto":"fullAuto",
+            "click #fullGround":"fullGround"
+        },
+        
+        sendall: function (event) { 
+            var autolevel=0;
+           
+            if (this.ui.head_select.val()==="Off")
+            {
+                
+            } else
+            {
+                autolevel = autolevel + Math.pow(2,9);
```

Comment:
http://stackoverflow.com/a/2803188/875806 and/or setting constants instead of re-computing every time (exponentiation is not a cheap operation in general).